app/models/foreman_tasks/lock.rb

```diff
@@ -1,5 +1,7 @@
 module ForemanTasks
   class Lock < ApplicationRecord
+    OWNER_LOCK_NAME = :task_owner
```

Comment:
Originally tasks were linked to users using locks. Later we swapped this for explicitly storing user_id on tasks, that's why the constant was needed in the failing migration. Once that migration is done, there should be no locks using `OWNER_LOCK_NAME` as a resource_type, meaning the use of this constant in the api controller makes no sense.